data/mn/legislature/Cal-Bahr-57b9021d-f5b7-4f80-a82d-a93ca9a123d0.yml

```diff
@@ -1,24 +1,30 @@
 id: ocd-person/57b9021d-f5b7-4f80-a82d-a93ca9a123d0
-name: Cal Bahr
+name: Calvin K. Bahr
 given_name: Cal
 family_name: Bahr
 gender: Male
-email: rep.cal.bahr@house.mn
-image: https://www.house.leg.state.mn.us/hinfo/memberimgls92/31B.gif?v=112221
+email: sen.cal.bahr@senate.mn
+image: https://www.senate.mn/img/member_thumbnails/31Bahr.jpg
 party:
 - name: Republican
 roles:
-- type: lower
+- end_date: '2023-01-03'
+  type: lower
   jurisdiction: ocd-jurisdiction/country:us/state:mn/government
   district: 31B
+- type: upper
+  jurisdiction: ocd-jurisdiction/country:us/state:mn/government
+  district: '31'
 offices:
 - classification: capitol
-  address: 321 State Office Building;St. Paul, MN 55155
-  voice: 651-296-2439
+  address: 2415 Minnesota Senate Bldg.; St. Paul, MN 55155
+  voice: 651-296-3219
 links:
 - url: http://www.house.leg.state.mn.us/members/profile/15455
+- url: https://www.senate.mn/members/member_bio.html?mem_id=1261
 other_identifiers:
-- scheme: legacy_openstates
+- scheme: legacy_openstatess
```

Comment:
@bfossen-ce extra s in legacy_openstates?

Reply:
@maxknutsen good catch. I have removed the extra s